### ISSUE TITLE
Mon: Hyperlink to ecal.ini

### ISSUE THREAD

**Issue by FlorianReimold** [continental/ecal#563]
People sometimes struggle to find and edit the `ecal.ini`. As the monitor shows the path to the file anyways, we could easily transfor it into a hyperlink the user can click on.

![grafik](https://user-images.githubusercontent.com/11774314/158797738-758a82c9-6988-46bd-a8d8-2f4861cb81a7.png)

**Comment by FlorianReimold** [comment]
Current solution already works fine. It however cannot open the `ecal.ini` on Linux as root. I want to do a workaround at some point:

1. Check the permissions

2. Query mime type:
    ```
    Shell: xdg-mime query filetype /usr/local/etc/ecal/ecal.ini
    C++: auto mime_type = QMimeDatabase().mimeTypeForFile(path);
    
    text/plain
    ```

3. Get Default application
    ```
    Shell: xdg-mime query default text/plain
    
    org.gnome.gedit.desktop
    ```

4. Unfortunatelly it stops here. But we can launch gedit with a magic parameter manually whenever we detect `org.gnome.gedit.desktop`:
    ```
    gedit admin:///usr/local/etc/ecal/ecal.ini
    ```

For all other cases we just let Qt open the file (it will use `xdg-open`). This would give us a generic solution:
- Gnome: Usually gedit is the default editor and our workaround is activated
- KDE: Usually Kate is the default editor. Kate asks the user itself to elevate the permissions when the file cannot be saved otherwise. So no action necessary.
- Fallback: The user can at least see the file, but will probably have to take care of editing it with sudo rights himself.

**Comment by FlorianReimold** [comment]
Fixed by #569
